### DISCUSSION TITLE
Whatsapp bot to convert media to sticker

### DISCUSSION THREAD

**Discussion by deven96** [tulir/whatsmeow#152] [Show and tell]
Was able to use this library to create a WhatsApp bot that automatically converts images/video to stickers
View it at https://github.com/deven96/whatsticker and chat with the bot at https://wa.me/19293792260

**Comment by baddiehubed** [comment]
A WhatsApp bot that converts media into stickers is an innovative and engaging solution for fashion websites. It allows images and short videos to be instantly transformed into attractive WhatsApp stickers. Fashion brands [Official Website](https://baddiehubed.com/)  and creators can share their outfits, logos, poses, and aesthetic designs as stickers, significantly boosting audience engagement. Such interactive tools not only increase brand visibility but also help users emotionally connect with the content — a crucial factor in today’s digital fashion ecosystem.
